### PR TITLE
Source collection fixes

### LINKED ISSUE #1177: Remove Collections page, redirect to Sources
We want to remove the Collections page:
https://search.creativecommons.org/collections

The Collections URL should be redirected to the forthcoming /sources page, which is already live in the dev environment:
https://ccsearch-dev.creativecommons.org/sources

We will also fix test and whitespace wrapping on the "sources" table here.

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Fixes #1177 and #1178 by @annatuma 

- Makes sources table sortable
- Fixes whitespace wrapping on sources table
- Removes '/collections' from code and navigation
- Redirects '/collections' to '/sources'
- Links button to correct new issue template in `creativecommons/cccatalog`
- Use proper i18n string for issue button contents

## Screenshots

Screen Shot 2020-08-24 at 6 43 10 AM
Screen Shot 2020-08-24 at 6 43 13 AM

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/pages/SourcePage.vue [hunk 1]
```diff
@@ -130,15 +151,38 @@ const SourcePage = {
...
   },
   methods: {
     getProviderImageCount(imageCount) {
       return imageCount.toLocaleString('en')
     },
+    sortTable(field) {
+      let direction = 'asc'
+      if (field === this.sort.field) {
+        direction = this.sort.direction === 'asc' ? 'desc' : 'asc'
+      }
```
Comment (zackkrida) [inline src/pages/SourcePage.vue:179]: here i only invert the sort direction if you're on the same column, otherwise 'reset' to the default, in this case 'asc'.

File: src/pages/SourcePage.vue [hunk 2]
```diff
@@ -148,4 +192,31 @@ export default SourcePage
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style lang="scss" scoped>
 @import '../styles/text-only-page.scss';
+
+.table.is-bordered {
+  th {
+    cursor: pointer;
+  }
+
+  // th:focus,
```
Comment (kgodey) [inline src/pages/SourcePage.vue:201]: Why not just remove this instead of commenting it out?

Comment (zackkrida) [reply src/pages/SourcePage.vue]: No reason; removed.
